index.js

```diff
@@ -10,8 +10,14 @@ module.exports = {
     },
   },
   rules: {
+    // The rule is enabled in Airbnb config, but is checked by the TypeScript compiler
+    // https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/default-props-match-prop-types.md
+    'react/default-props-match-prop-types': 'off',
     // Append 'tsx' to Airbnb 'react/jsx-filename-extension' rule
     // https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-filename-extension.md
     'react/jsx-filename-extension': ['error', { extensions: ['.jsx', '.tsx'] }],
+    // The rule is enabled in Airbnb config, but is checked by the TypeScript compiler
+    // https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/prop-types.md
+    'react/prop-types': 'off',
```

Comment:
These rules also apply to JavaScript files, and we'd want to keep the rules enabled for that

Also some people like to use prop types with TypeScript for runtime type checking

It'd be good to revert these changes, I'm not sure they belong within this config

Reply:
Got it! Reverted!